src/flows/pipes/MetricSelector/index.ts

```diff
@@ -11,6 +11,7 @@ export default register => {
     component: View,
     readOnlyComponent: ReadOnly,
     button: 'Metric Selector',
+    disabled: true,
```

Comment:
What's this do?

Reply:
just hides it from the user without removing it from the code. it'll be back and i still like using it, so........